### DISCUSSION TITLE
.NET Framework requirement

### DISCUSSION THREAD

**Discussion by Hardexit** [PSAppDeployToolkit/PSAppDeployToolkit#1108] [Q&A] [answered]
If I understand the line correctly, is it possible to bypass this dependency?
https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/blob/7ff49b07103b78ac945cfc0df012384d6fb5578c/src/PSAppDeployToolkit/PSAppDeployToolkit.psd1#L41

**Comment by mjr4077au** [comment]
It's not possible, our DLL files require this version at an absolute minimum, and its also the oldest version supported by Microsoft.

What version of .NET Framework are you attempting to target?

**Reply by Hardexit** [reply]
So currently 3.1.32, 4.7.2 and 4.8.1 are installed on the computers.

**Reply by mjr4077au** [reply, marked as answer]
The version number in the manifest is a minimum, not a requirement. Anything .NET 3.x is absolutely out of the question, however the toolkit should work fine with your 4.7.2 and 4.8.1 targets.
